TodoListWebApp/appsettings.json

```diff
@@ -1,10 +1,18 @@
 ﻿{
   "AzureAd": {
+    // Coordinates of the Azure AD Tenant
     "Instance": "https://login.microsoftonline.com/",
     "Domain": "[Enter the domain of your tenant, e.g. contoso.onmicrosoft.com]",
     "TenantId": "[Enter the Tenant Id (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+
+    // Coordinates of the TodoListWebApp
     "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
-    "CallbackPath": "/signin-oidc"
+    "CallbackPath": "/signin-oidc",
+    "ClientSecret": "[Enter the Client Secret (Application password) added from the Keys section for the application in the Azure portal]",
+
+    // Coordinates of the TodoListService
+    "TodoListResourceId": "[Enter the Client Id (Application ID) of the TodoListService, obtained from the Azure portal), e.g. 11111111-1111-1111-11111111111111111]", // ClientId of the TodoList Service
+    "TodoListBaseAddress": "https://localhost:44351"
   },
```

Comment:
The appsettings.json for the ASP.NET application got enriched with:
- the client secret of the TodoList Web App, which is necessary for the confidential client flow to get a token for the TodoListService Web API
- the coordinates of the TodoListService (resourceId and base URL)